ServerlessLibraryAPI/LibraryController.cs

```diff
@@ -26,11 +27,27 @@ public LibraryController(ICacheService cacheService)
         // GET: api/<controller>
         [HttpGet]
         [ProducesResponseType(typeof(IEnumerable<LibraryItem>), 200)]
-        public JsonResult Get()
+        public JsonResult Get(string filterText, string type, string language)
```

Comment:
It would be better if we can get the filtered results at the client side itself. 
We should avoid round-tripping if possible.

Reply:
We will consider this after getting requirement from portal